documentation/build/html/_sources/changelog.txt

```diff
@@ -0,0 +1,58 @@
+*********
```

Comment:
Either make /README.md use this file or use the contents from there here. Having the same information duplicated is no good.